vocabularies/quality.ttl

```diff
@@ -0,0 +1,685 @@
+# baseURI: https://linked.data.qld.gov.au/vocab
+# imports: https://linked.data.qld.gov.au/def/dcat
+
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix dq-iso-19157: <https://linked.data.gov.au/def/ISO-19157/quality-dimension/> .
+@prefix dq-iso-25012: <https://linked.data.gov.au/def/ISO-IEC-25012/quality-dimension/> .
+@prefix dq-nsw: <https://linked.data.gov.au/def/NSW-OEH/quality-dimension/> .
+@prefix dqv: <http://www.w3.org/ns/dqv#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix sdo: <http://www.schema.org/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+dq-iso-19157:Dimensions
+  a skos:ConceptScheme ;
+  dcterms:source "ISO 19157"@en ;
+  rdfs:label "Dimensions of data quailty (ISO 19157)"@en ;
+  skos:prefLabel "Dimensions of data quailty (ISO 19157)"@en ;
+  dcterms:created "2020-11-16"^^xsd:date ;
+  dcterms:creator <http://linked.data.gov.au/def/gsq-alias/des> ;
+dcterms:modified "2020-11-16"^^xsd:date ;
+dcterms:publisher <http://linked.data.gov.au/def/gsq-alias/des> ;
+  skos:hasTopConcept dq-iso-19157:Completeness ;
+  skos:hasTopConcept dq-iso-19157:LogicalConsistency ;
+  skos:hasTopConcept dq-iso-19157:Metaquality ;
+  skos:hasTopConcept dq-iso-19157:PositionalAccuracy ;
+  skos:hasTopConcept dq-iso-19157:TemporalQuality ;
+  skos:hasTopConcept dq-iso-19157:ThematicAccuracy ;
+  skos:hasTopConcept dq-iso-19157:UsabilityElement .
+
+dq-iso-19157:AbsoluteExternalPositionalAccuracy
+  a dqv:Dimension ; 
+  a skos:Concept ;
+  rdfs:label "AbsoluteExternalPositionalAccuracy"@en ;
+  skos:broader dq-iso-19157:PositionalAccuracy ;
+  skos:definition "Closeness of reported coordinate values to values accepted as or being true"@en ;
+  skos:inScheme dq-iso-19157:Dimensions ;
+  skos:prefLabel "Absolute External Positional Accuracy"@en .
+
+dq-iso-19157:AccuracyOfATimeMeasurement
+  a dqv:Dimension ; 
+  a skos:Concept ;
+  rdfs:label "AccuracyOfATimeMeasurement"@en ;
+  skos:broader dq-iso-19157:TemporalQuality ;
+  skos:definition "Correctness of the temporal references of an item (reporting of error in time measurement)"@en ;
+  skos:inScheme dq-iso-19157:Dimensions ;
+  skos:prefLabel "Accuracy of a Time Measurement"@en .
+
+dq-iso-19157:Completeness
+  a dqv:Dimension ; 
+  a skos:Concept ;
+  rdfs:label "Completeness"@en ;
+  skos:definition "Presence and absence of features, their attributes and their relationships"@en ;
+  skos:inScheme dq-iso-19157:Dimensions ;
+  skos:prefLabel "Completeness"@en .

```

Comment:
There are two concepts called Completeness.
These should be amalgamated if they are the same, or given unique labels that distinguish them from each other if not.

Reply:
They are technically a bit different, one is sourced in ISO25012 the other is in ISO19157. While the definitions are similar, since from different sources we probably want to keep them separate.

Based on that do we need to do anything here?

We are still discussing the best way of recording quality, so we could end up with a collection to limit the terms. Have more internal discussion to occur.

Reply:
In this instance some sort of distinction should be made. Even if it is _Completeness (ISO25012)_ and _Completeness (ISO19157)_.

Overlapping labels create issues with drop downs, automated scripts that leverage vocabularies, etc Ambiguous choices are difficult for humans and nigh on impossible for machines.
https://github.com/cmader/qSKOS/wiki/Quality-Issues#Overlapping_Labels

Its not essential in that it won't 'break' the file. But it is highly recommended